composer.json

```
@@ -25,45 +25,45 @@
         "ext-dom": "*",
         "ext-json": "*",
         "ext-sqlite3": "*",
-        "acquia/coding-standards": "^0.7.0",
-        "composer/composer": "~2.2.5",
+        "acquia/coding-standards": "^0.8.0",
```

Oops. This is probably technically a backward-compatibility break that will need communication beforehand. Would you create an issue in our backlog to do the communication and then the update? In the meantime, lets drop this back down.

```suggestion
        "acquia/coding-standards": "^0.7.0",
```